command-snapshot.json

```diff
@@ -1,31 +1,4 @@
 [
-    {
-        "command": "commandreference:generate",
-        "flags": [
-            "json",
-            "loglevel",
-            "outputdir",
-            "plugins",
-            "hidden",
-            "erroronwarnings"
-        ]
-    },
-    {
-        "command": "commands",
-        "flags": [
-          "help",
-          "json",
-          "hidden",
-          "columns",
-          "sort",
-          "filter",
-          "csv",
-          "output",
-          "extended",
-          "no-truncate",
-          "no-header"
-      ]
-    },
```

Comment:
In a previous version of plugin-command-snapshot these were included by default but they no longer are.  This always should have been just the commands in this plugin so this is a good thing.  No features are being removed from this plugin.